.github/workflows/code_quality.yml

```
@@ -0,0 +1,20 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2.3.0
+        with:
+          flutter-version: '3.5.0'
+      - name: Install packages
+        run: flutter pub get
+      - name: Linter
+        run: flutter analyze
+      - name: Test
+        run: flutter test
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)